README.md

```diff
@@ -1 +1,84 @@
-# ract-redux-capstone
\ No newline at end of file
+# REACT-REDUX-CAPSTONE
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+We will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages.
+
+## Screenshots
+
+- Home Page
+![screenshot](./app_screenshot.png)
+- Details Page
+![screenshot](./app_screenshot1.png)
+
+
+## Video
```

Comment:
Please record a new video  and highlight about a piece of the code that you find interesting or is proud of making. You do not need to present the whole project again but just talk about the piece of code which you are proud of.

Reply:
thanks